static.routes.js

```diff
@@ -36,27 +36,82 @@ export default async () => {
     })),
 
     // Routes for components, i.e. /element/button
-    ..._.map(getComponentMenu(), (baseInfo) => ({
-      path: getComponentPathname(baseInfo),
+    ..._.map(
+      _.filter(getComponentMenu(), (baseInfo) => !baseInfo.external),
+      (baseInfo) => ({
+        path: getComponentPathname(baseInfo),
+        component: 'docs/src/components/ComponentDoc',
+        priority: 0.8,
+        getData: async () => {
+          const componentsInfo = getComponentGroupInfo(baseInfo.displayName)
+          const sidebarSections = getSidebarSections(baseInfo.displayName)
+
+          return {
+            componentsInfo,
+            exampleSources,
+            sidebarSections,
+            displayName: baseInfo.displayName,
+            deprecated: !!_.find(
+              _.get(componentsInfo[baseInfo.displayName], 'docblock.tags'),
+              (tag) => tag.title === 'deprecated',
+            ),
+            seeTags: getInfoForSeeTags(componentsInfo[baseInfo.displayName]),
+          }
+        },
+      }),
+    ),
+
+    {
+      path: `/addons/ref/`,
       component: 'docs/src/components/ComponentDoc',
       priority: 0.8,
       getData: async () => {
-        const componentsInfo = getComponentGroupInfo(baseInfo.displayName)
-        const sidebarSections = getSidebarSections(baseInfo.displayName)
+        const componentsInfo = {
```

Comment:
It's annoying to do this manually, but in any case this is not a long term solution